src/app/event/new-scheduled-event/new-scheduled-event.component.html

```diff
@@ -349,7 +349,7 @@ <h3>Simple Mode Compatibility</h3>
     </ng-container>
   </clr-wizard-page>
 
-  <clr-wizard-page>
+  <clr-wizard-page [clrWizardPageNextDisabled]="(selectedcourses.length == 0 && selectedscenarios.length == 0) || selectedEnvironments.length == 0 || !vmCounts.valid || !simpleModeVmCounts.valid || (se.start_time >= se.end_time) || !se.start_time || !se.end_time || !eventDetails.valid">
```

Comment:
Is it possible we make this calculated in the typescript? this is sort of... big.

Reply:
ideally using if/else so it's easier to understand for the next dev

Reply:
Yep i will create a method for this check.